src/Microsoft.Identity.Web/AzureFunctionsAuthenticationHttpContextExtension.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Extensions for <see cref="AzureFunctionsAuthenticationHttpContextExtension"/>.
+    /// </summary>
+    public static class AzureFunctionsAuthenticationHttpContextExtension
+    {
+        /// <summary>
+        /// Enables Bearer authentication for an API for use in Azure Functions.
```

Comment:
What about?
Enables an Azure Function to act as/expose a protected web API, enabling bearer token authentication? Calling this method from your Azure function validates the token and exposes the identity of the user or app on behalf of which your function is called, in the `HttpContext.User` member, where your function can exploit it.

Reply:
@jmprieur what do you mean by add the test app?

Reply:
Hmm PR is approved, should I still go ahead and edit this description? New PR or same?

Reply:
@maliksahil go ahead and edit. as it's a fork, @jmprieur  or i will have to merge it. thanks.

Reply:
Done!! :-)

Reply:
@jennyf19, I meant the sample using the new API: https://github.com/maliksahil/ms-identity-azurefunctions-microsoft-identity-web